src/ensembl/src/content/app/browser/browser-bar/BrowserBar.test.tsx

```diff
@@ -52,25 +49,20 @@ describe('<BrowserBar />', () => {
       expect(renderedBrowserBar.find(BrowserLocationIndicator).length).toBe(1);
     });
 
-    test('renders GeneSummaryStrip if focus object is gene', () => {
-      const renderedBrowserBar = mount(
-        renderBrowserBar({ ensObject: createEnsObject('gene') })
-      );
-      expect(renderedBrowserBar.find(GeneSummaryStrip).length).toBe(1);
-    });
-
-    test('renders RegionSummaryStrip if focus object is region', () => {
-      const renderedBrowserBar = mount(
-        renderBrowserBar({ ensObject: createEnsObject('region') })
-      );
-      expect(renderedBrowserBar.find(RegionSummaryStrip).length).toBe(1);
+    test('contains FeatureSummaryStrip', () => {
+      expect(renderedBrowserBar.find(FeatureSummaryStrip).length).toBe(1);
```

Comment:
@azangru  just making sure this is still valid if i change the if check to ternary.

Reply:
Totally valid :-) The test would have failed otherwise.

Reply:
Thanks @azangru i have push the ternary change, please have a look and let me know if its all good to merge. I have another PR waiting :)